### ISSUE TITLE
Post Entry Text Area grows vertically

### ISSUE THREAD

**Issue by alexbevi** [alexclemensua/scummvm-phpBB3-theme#2]
Not sure if this is something that can be addressed, but it's a bit odd. Tested using Chrome Beta on Linux Mint 17.2

Original:

![image](https://cloud.githubusercontent.com/assets/135803/11811321/d1aa2d7e-a302-11e5-9d21-38c9631e8426.png)

Pressed ALT a bunch of times:

![image](https://cloud.githubusercontent.com/assets/135803/11811360/0d1713cc-a303-11e5-97a3-97af5fe16284.png)

**Comment by DrMcCoy** [comment]
Yes, also happens here with Iceweasel (Firefox) 42.0 on Debian GNU/Linux.
